### DISCUSSION TITLE
Pyodide 0.26.2 Offline Install HTTPS Server TLS Browser Support?

### DISCUSSION THREAD

**Discussion by juparker37** [pyodide/pyodide#4991] [General]
Is it possible to run Pyodide 0.26.2 offline via HTTPS TLS in the browser? Does HTTPS need the SSL package? Firefox shows https://vm_name:9450/ as unsecure. I could not find any options on the http.server -h options.

**Comment by ryanking13** [comment]
Well, if you set a proper HTTPS server offline, it will work. Pyodide is just a list of files that can be served through the file server. For instance, [JsDelivr CDN](https://cdn.jsdelivr.net/pyodide/v0.26.2/full/pyodide.js) which we put the Pyodide releases also requires HTTPS, so there is no limit in using HTTPS.
